### ISSUE TITLE
Error during configuration

### ISSUE THREAD

**Issue by indiejames** [tom-sherman/unison-vscode-ui#19]
I get the following error when trying to configure the extension:
````
Unexpected token E in JSON at position 0
```
The url I'm trying to use is
```
http://127.0.0.1:64564/GGMdX_sRqdoVBKY2/api
```
Which is the url of the API for my local codebase (ucm is running and I can open the UI).

If I had to guess I would say that the unexpected 'E' is probably the start of en error message coming from the API when the plugin is expecting JSON.

**Comment by timjs** [comment]
Having the same problem here. Anybody an idea how to fix this?

**Comment by tom-sherman** [comment]
What unison version do you folks have? It's likely a change to the API.

**Comment by tom-sherman** [comment]
Should be fixed in [v0.1.4](https://github.com/tom-sherman/unison-vscode-ui/releases/tag/v0.1.4)
